packages/manager/modules/pci/src/projects/project/ai-dashboard/onboarding/translations/Messages_fr_FR.json

```diff
@@ -1 +1,23 @@
-{}
+{
+  "pci_ai_dashboard_onboarding_tuto": "Tutoriels",
+  "pci_ai_dashboard_onboarding_tuto_notebook_title": "Créer un AI Notebook",
+  "pci_ai_dashboard_onboarding_tuto_notebook_description": "Lorem ipsum dolor, sit amet consectetur adipisicing elit. Eligendi saepe cumque odit.",
+  "pci_ai_dashboard_onboarding_tuto_training_title": "Créer un AI Training",
+  "pci_ai_dashboard_onboarding_tuto_training_description": "Lorem ipsum dolor, sit amet consectetur adipisicing elit. Eligendi saepe cumque odit.",
```

Comment:
**question:** Is it a valid description?

Reply:
Good catch, should have been removed before. Done here: https://github.com/ovh/manager/pull/8160/commits/e3e42f50be6eb5aece2f0c8b17e28c9c2f03ccb4